src/UI/InfoController.java

```diff
@@ -0,0 +1,19 @@
+package UI;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.CommanderInfo.InfoPage;
+import Engine.GameInstance;
+import Engine.IController;
+
+public interface InfoController extends IController
```

Comment:
"InfoController" seems unnecessarily non-specific.  Maybe CommanderInfoController or something would convey intent better. I know there's a lot of dependent pull requests, so I'm probably fine putting this in the "address later" bin.

Reply:
I was leaving it generic because there's no reason we couldn't use it for something else later.
It's just a paginated controller with relatively generic inputs.

Reply:
Fair enough. I might buy that. Then again, it's got `getSelectedCO` and `getSelectedCOInfo`, which would make using it for non-CO-related things awkward, at best.

Reply:
Pretty sure at least `getSelectedCO` is allowed to return `null`. It's no worse than `StateData`.

Reply:
And now I need to decide if that means "Oh, this is fine then" or "Maybe I also need to worry about `StateData`".
Haha. I think.